scripts/dockerfiles/Dockerfile-base

```diff
@@ -34,6 +34,9 @@ RUN useradd -m --comment 'hcf user' vcap && \
 
 ADD monitrc.erb /opt/hcf/monitrc.erb
 
+ADD post-start.sh /opt/hcf/post-start.sh
+RUN chmod ug+x /opt/hcf/post-start.sh
```

Comment:
Any particular reason this isn't `a+x`?
(If we ever end up running `run.sh` as `vcap` it will be the wrong group, as IIRC this is owned by `root/root`.)